src/System.Private.CoreLib/src/System/Diagnostics/Eventing/RuntimeEventSource.cs

```diff
@@ -0,0 +1,143 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+
+using System.Runtime.InteropServices;
+using Microsoft.Win32;
+using Microsoft.Win32.SafeHandles;
+
+using System.Runtime.InteropServices.ComTypes;
```

Comment:
Do you really need all these namespaces?

Reply:
I think these got pulled in from my branch where I was using these namespaces. I'll remove them. Thanks!